libraries/bot-integration-tests/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "bot-integration-tests",
+  "version": "4.1.6",
+  "description": "Integration tests for botbuilder",
+  "scripts": {
+    "build": "tsc",
+    "just-test": "tsc && mocha \"./transpiled-tests/**/*.test.js\"",
+    "test": "tsc && nyc mocha \"./transpiled-tests/**/*.test.js\"",
+    "set-version": "npm version --allow-same-version ${Version}"
+  },
+  "author": "Microsoft Corp.",
+  "license": "MIT",
+  "dependencies": {
+    "botbuilder": "4.1.6",
+    "botframework-connector": "4.1.6"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.7",
+    "@types/mocha": "^5.2.7",
+    "@types/restify": "^8.4.2",
+    "express": "^4.17.1",
+    "mocha": "^6.2.3",
+    "nyc": "^15.1.0",
+    "restify": "^8.5.1"
+  }
```

Comment:
Hi @stevengum, we didn't add it because we found that it's not usually specified on other libraries but we'll add it now. We'll add to work with v3.5.3 (as some of the libraries have).